sympy/matrices/eigen.py

```diff
@@ -838,16 +838,16 @@ def _is_positive_semidefinite_minors(M):
 
 
 def _jordan_form(M, calc_transform=True, **kwargs):
-    """Return ``(P, J)`` where `J` is a Jordan block
-    matrix and `P` is a matrix such that
+    """Return ``(P, J)`` where ``J`` is a Jordan block
+    matrix and ``P`` is a matrix such that
 
         ``M == P*J*P**-1``
```

Comment:
I'd also change these to use LaTeX than python scripts because they are more about math and LaTeX wouldn't look longer than python.